.github/workflows/issue-triage.yml

```diff
@@ -0,0 +1,43 @@
+name: Auto-triage issues to project boards
+
+on:
+  issues:
+    types: [opened, labeled]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  COLUMN_NAME: Needs Triage # column where issues will be created
+
+jobs:
+  content_project:
+    runs-on: ubuntu-latest
+    env:
+      PROJECT_ID: 11873277
+      PROJECT_NAME: Docs PRs and Issues
+    name: Triage to Content Project
+    steps:
+    - name: Triages NEW issues to the Content Project
+      if: github.event.action == 'opened'
+      run: |
+        HEADER="Accept: application/vnd.github.inertia-preview+json"
+        ISSUE_ID=${{ github.event.issue.id }}
+        COLUMN=$(gh api -H "$HEADER" projects/$PROJECT_ID/columns --jq ".[] | select(.name == \"$COLUMN_NAME\").id")
+        gh api -H "$HEADER" -X POST projects/columns/$COLUMN/cards -f content_type='Issue' -F content_id=$ISSUE_ID
+
+  localization_project:
+    runs-on: ubuntu-latest
+    env:
+      PROJECT_ID: 12004783
+      PROJECT_NAME: Docs Site Localization PRs and Issues
+    name: Triage to Localization Project
+    steps:
+    - name: Triages NEW issues and NEW pull requests to the Localization Project
```

Comment:
i think you can update the name for this now

Reply:
Good catch! I have updated it